api/base/settings.py

```diff
@@ -41,13 +41,14 @@
 
     # 3rd party
     'rest_framework',
+    'rest_framework_swagger',
 )
 
 REST_FRAMEWORK = {
     'PAGE_SIZE': 10,
     'DEFAULT_RENDERER_CLASSES': (
-        'api.base.renderers.JSONAPIRenderer',
         'rest_framework.renderers.JSONRenderer',
+        'api.base.renderers.JSONAPIRenderer',
```

Comment:
As @samanehsan and I discovered yesterday, order here is important due to what seems to be a bug in rest_framework_swagger. Can you add a comment here explaining this? Bonus points for linking to the issue, if you've posted one. =)